$NetBSD$

# Add NetBSD support - just exclude jogshuttle similar to apple/FreeBSD

--- src/CMakeLists.txt.orig	2010-09-14 18:53:22.000000000 +0000
+++ src/CMakeLists.txt
@@ -224,11 +224,11 @@ add_subdirectory( ${CMAKE_SOURCE_DIR}/sr
 
 add_definitions( ${KDE4_DEFINITIONS} )
 
-if(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR NO_JOGSHUTTLE)
+if(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "NetBSD" OR NO_JOGSHUTTLE)
   add_definitions(-DNO_JOGSHUTTLE)
-else(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR NO_JOGSHUTTLE)
+else(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "NetBSD" OR NO_JOGSHUTTLE)
   set(kdenlive_SRCS jogshuttle.cpp ${kdenlive_SRCS})
-endif(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR NO_JOGSHUTTLE)
+endif(APPLE OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "NetBSD" OR NO_JOGSHUTTLE)
 
 if (APPLE)
   set(kdenlive_SRCS videoglwidget.cpp ${kdenlive_SRCS})
